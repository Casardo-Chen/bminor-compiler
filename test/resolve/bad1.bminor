// conflicting declaration
x: integer;
y: boolean;
z: void;
s: char;
t: string;
a: integer = 500000;
a: char = "this is a test";
z: float = 3E2;
b: boolean = true;