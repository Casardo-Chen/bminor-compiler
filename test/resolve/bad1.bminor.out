resolve error: duplicate definitions of a in the same scope is invalid.
resolve error: duplicate definitions of z in the same scope is invalid.
2 resolve errors found.
